web-app/src/pages/tournament/TournamentCreator.vue

```diff
@@ -191,22 +191,21 @@ export default class TournamentCreator extends Vue {
       align: 'left',
       field: 'name',
       sortable: true,
-      classes: 'textWrapDotted',
-      style: 'max-width: 8em',
     },
     {
       name: 'participants',
+      required: true,
```

Comment:
Is `required` is  required? 
In doc `required` has desc "If we use visible-columns, this col will always be visible".
I think, You are not using `visible-columns`